core/src/main/java/org/jdbi/v3/Cleanables.java

```diff
@@ -35,120 +35,17 @@ private Cleanables()
 
     static Cleanable forResultSet(final ResultSet rs)
     {
-        return new ResultSetCleanable(rs);
+        return () -> rs.close();
```

Comment:
Meh: you could use `rs::close` instead of a lambda. They're semantically equivalent--I just like the way method handles look.

Not critical for merge, obviously.. :)